webapp/src/views/organizations/sso/OrganizationSsoView.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
+import { useTranslate } from '@tolgee/react';
+import { BaseOrganizationSettingsView } from '../components/BaseOrganizationSettingsView';
+import { LINKS, PARAMS } from 'tg.constants/links';
+import { useOrganization } from '../useOrganization';
+import { CreateProviderSsoForm } from 'tg.views/organizations/sso/CreateProviderSsoForm';
+import { useApiQuery } from 'tg.service/http/useQueryApi';
+import { FormControlLabel, Switch } from '@mui/material';
+import Box from '@mui/material/Box';
+
+export const OrganizationSsoView: FunctionComponent = () => {
+  const organization = useOrganization();
+  const { t } = useTranslate();
+  if (!organization) {
+    return null;
+  }
+
+  const providersLoadable = useApiQuery({
+    url: `/v2/{organizationId}/sso/providers`,
+    method: 'get',
+    path: {
+      organizationId: organization.id,
+    },
+  });
+
+  const credentialsRef = useRef({
```

Comment:
Is there any special reason to use useRef? IMO, you can just pass the loadable and handle it all in the Form class by setting initial values. This logic can be pretty much copied from other pars of the system.